src/jquery.quicksearch.js

```diff
@@ -177,7 +184,12 @@
 			}
 			
 			if (noresults) {
-				this.results(false);
+				if(options.onNoResultsFound !== null){
+					options.onNoResultsFound();
+				}else{
+					this.results(false);
+				}
+				
```

Comment:
Ah good point. Let's make it:

```js
if ($.isFunction(options.onNoResultsFound)) {
   options.onNoResultsFound(this);
}
else {
   this.results(false);
}
```

Notice the this param in your callback... That way, one could still call this.results(false) in their callback :P

Simply add commits to this PR ! Thanks!